source/fe/fe_values.cc

```diff
@@ -473,15 +473,19 @@ namespace FEValuesViews
       AssertDimension (values.size(), n_quadrature_points);
 
       std::fill (values.begin(), values.end(),
-                 dealii::internal::NumberType<typename std::decay<Number>::type>::value(0.0));
+                 dealii::internal::NumberType<Number>::value(0.0));
```

Comment:
@bangerth This changes attends to your concern that you raised in [this comment](https://github.com/dealii/dealii/pull/4951/files#r135063037). If this patch is accepted then I consider my obligation to answer your follow-up request, which I've sheepishly ignored, nullified ;-)

Reply:
I appreciate it ;-)